packages/checkouts/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@thirdweb-dev/checkouts",
+  "version": "1.3.3",
```

Comment:
question if we continue with `paperxyz` versions or start fresh from 1.0.0 since this is technically a new package

Reply:
Starting fresh on /payments